Winapp2.ini

```diff
@@ -3957,8 +3957,12 @@ DetectFile2=%CommonAppData%\Asus
 DetectFile3=%ProgramFiles%\ASUS\AI Suite
 FileKey1=%AppData%\ASUS WebStorage\Logs|*.*
 FileKey2=%CommonAppData%\Asus\AsusAppStore|*.log|RECURSE
-FileKey3=%ProgramFiles%\ASUS\*|*.log;*log*.txt;*.tmp|RECURSE
-FileKey4=%SystemDrive%|wifi.log
+FileKey3=%CommonAppData%\Asus\ARMOURY CRATE Service|*.log|RECURSE
+FileKey4=%CommonAppData%\Asus\ArmouryCrate|*.log|RECURSE
+FileKey5=%CommonAppData%\Asus\ALL|*.log|RECURSE
+FileKey6=%CommonAppData%\Asus\SCD|*.log|RECURSE
+FileKey7=%ProgramFiles%\ASUS\*|*.log;*log*.txt;*.tmp|RECURSE
+FileKey8=%SystemDrive%|wifi.log

```

Comment:
Couldn't the new FileKeys be merged?

FileKey3=%CommonAppData%\\Asus\\\*|\*.log|RECURSE

Reply:
Sry not possible. It will delet the install Log and it makes some problems with the deinstallation process with AI Suite (Old Problem).